Fundamentals 1/exercises/string1.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <title>Strings: Task 1</title>
+    <style>
```

Comment:
Why don't you add these styles to the .styles.css?

Reply:
Because I wanted to do it like we did in the class, inside the HTML. Just to try it. The rest of my assignment, the css.style is external.